src/test/kotlin/day05/PersonTest.kt

```
@@ -0,0 +1,13 @@
+package day05
+
+import org.junit.Test
+
+import org.junit.Assert.*
+ class PersonTest {
+
+@Test
+ fun testToString() {}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)